<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="FullWiedgit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="FullWiedgitPage1" native="true">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1091</width>
        <height>641</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-raduis:20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Pictures/newGym.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>240</y>
        <width>261</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    background-color: rgba(255, 255, 255, 0.75);
    border-radius: 40px;
    padding: 20px;</string>
      </property>
      <widget class="QLabel" name="LabelSubHead">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>-10</y>
         <width>151</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Yeasty Flavors&quot;;
background-color: rgba(255, 255, 255, 0);
color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEditUserName">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">    padding: 10px;
    border: 1px solid #ccc;
background-color: rgb(198, 143, 59);
font: 12pt &quot;Yeasty Flavors&quot;;
    border-radius: 10px;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEditPassword">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">    padding: 10px;
    border: 1px solid #ccc;
background-color: rgb(198, 143, 59);
font: 12pt &quot;Yeasty Flavors&quot;;
    border-radius: 10px;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>210</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(198, 143, 59);
    padding: 10px;
color: rgb(0, 0, 0);
font: 9pt &quot;Yeasty Flavors&quot;;
    border: none;
    border-radius: 25px;
hover background-color: #008FC1;</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLabel" name="LabelUserName">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>40</y>
         <width>131</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yeasty Flavors</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Yeasty Flavors&quot;;
background-color: rgba(255, 255, 255, 0);
color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>UserName :</string>
       </property>
      </widget>
      <widget class="QLabel" name="LabelPassword">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>100</y>
         <width>131</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yeasty Flavors</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Yeasty Flavors&quot;;
background-color: rgba(255, 255, 255, 0);
color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>Password :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>If you don't have account ,</string>
       </property>
      </widget>
      <widget class="QPushButton" name="toggleButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 85, 0);
background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Sign up</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="FullWiedgit_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1081</width>
        <height>631</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1091</width>
        <height>631</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(249, 234, 205);
border-raduis:20px;</string>
      </property>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>171</width>
         <height>611</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Yeasty Flavors&quot;;
color: rgba(0, 0, 0, 150);</string>
        </property>
        <property name="text">
         <string>Main Menu</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Dashboardbtn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>140</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 234, 205);
    padding: 10px;
color: rgb(0, 0, 0);
font: 9pt &quot;Yeasty Flavors&quot;;
    border: none;
    border-radius: 25px;
hover background-color: #008FC1;</string>
        </property>
        <property name="text">
         <string>Dashboard</string>
        </property>
       </widget>
       <widget class="QPushButton" name="GymMangbtn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>170</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 234, 205);
    padding: 10px;
color: rgb(0, 0, 0);
font: 9pt &quot;Yeasty Flavors&quot;;
    border: none;
    border-radius: 25px;
hover background-color: #008FC1;</string>
        </property>
        <property name="text">
         <string>Gym Mangment</string>
        </property>
       </widget>
       <widget class="QPushButton" name="TMbtn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>210</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 234, 205);
    padding: 10px;
color: rgb(0, 0, 0);
font: 9pt &quot;Yeasty Flavors&quot;;
    border: none;
    border-radius: 25px;
hover background-color: #008FC1;</string>
        </property>
        <property name="text">
         <string>Trainer Mangment</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Aubtn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>240</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 234, 205);
    padding: 10px;
color: rgb(0, 0, 0);
font: 9pt &quot;Yeasty Flavors&quot;;
    border: none;
    border-radius: 25px;
hover background-color: #008FC1;</string>
        </property>
        <property name="text">
         <string>Add Users</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Dashboard_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>280</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 234, 205);
    padding: 10px;
color: rgb(0, 0, 0);
font: 9pt &quot;Yeasty Flavors&quot;;
    border: none;
    border-radius: 25px;
hover background-color: #008FC1;</string>
        </property>
        <property name="text">
         <string>Admin</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Notibtn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>310</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 234, 205);
    padding: 10px;
color: rgb(0, 0, 0);
font: 9pt &quot;Yeasty Flavors&quot;;
    border: none;
    border-radius: 25px;
hover background-color: #008FC1;</string>
        </property>
        <property name="text">
         <string>Notification</string>
        </property>
       </widget>
       <widget class="QLabel" name="TrainerMangmentIcon">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/group-of-businessmen-svgrepo-com.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="DashBoardIcon">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/dashboard-svgrepo-com.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="GymManmentIcon">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/gym-dumbbell-svgrepo-com.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="AddUserIcon">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/add-user-svgrepo-com.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="AdminIcon">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/user-admin-svgrepo-com.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="NotificattionIcon">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>330</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/notification-bell-1397-svgrepo-com.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="Billbtn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>360</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 234, 205);
    padding: 10px;
color: rgb(0, 0, 0);
font: 9pt &quot;Yeasty Flavors&quot;;
    border: none;
    border-radius: 25px;
hover background-color: #008FC1;</string>
        </property>
        <property name="text">
         <string>Billing</string>
        </property>
       </widget>
       <widget class="QLabel" name="BillingIcon">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>370</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/invoice-bill-svgrepo-com.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="BillingIcon_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>450</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/profile-round-1342-svgrepo-com.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="ProfileBtn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>440</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 234, 205);
    padding: 10px;
color: rgb(0, 0, 0);
font: 9pt &quot;Yeasty Flavors&quot;;
    border: none;
    border-radius: 25px;
hover background-color: #008FC1;</string>
        </property>
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
       <widget class="QLabel" name="BillingIcon_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>480</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/setting-4-svgrepo-com.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="SettingBtn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>480</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 234, 205);
    padding: 10px;
color: rgb(0, 0, 0);
font: 9pt &quot;Yeasty Flavors&quot;;
    border: none;
    border-radius: 25px;
hover background-color: #008FC1;</string>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
       <widget class="QLabel" name="BillingIcon_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>520</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/logout-svgrepo-com.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="LogOutBtn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>520</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 234, 205);
    padding: 10px;
color: rgb(0, 0, 0);
font: 9pt &quot;Yeasty Flavors&quot;;
    border: none;
    border-radius: 25px;
hover background-color: #dcdcdc;
</string>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>410</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgba(0, 0, 0, 150);
font: 14pt &quot;Yeasty Flavors&quot;;</string>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
       <widget class="QLabel" name="logo">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>91</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/ChatGPT Image Apr 15, 2025, 10_53_31 PM.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>891</width>
         <height>611</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QWidget" name="widget_5" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>891</width>
           <height>611</height>
          </rect>
         </property>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>200</y>
            <width>211</width>
            <height>101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Dashboard</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <widget class="QWidget" name="widget_6" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>901</width>
           <height>611</height>
          </rect>
         </property>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>170</y>
            <width>291</width>
            <height>101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Train Mangment</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QWidget" name="widget_4" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>891</width>
           <height>611</height>
          </rect>
         </property>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>170</y>
            <width>211</width>
            <height>101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Gym Mangment</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
